lib/pythontarget.c

```diff
@@ -271,7 +320,7 @@ char** _lf_py_parse_argv_impl(PyObject* py_argv, size_t* argc) {
     char** argv;
 
     // Read the optional argvs
-    PyObject* py_argv_parsed;
+    PyObject* py_argv_parsed = NULL;
```

Comment:
I realized that the segfault I have been getting was due to not initializing py_argv_parsed and it having some random value on the stack. When PyListSize gets called on uninitialized py_argv_parsed later in the same function, it produces a segfault

Reply:
Nice catch!